OpenMEEG/libs/OpenMEEGMaths/include/BlasLapackImplementations/OpenMEEGMathsMKLConfig.h

```diff
@@ -10,7 +10,12 @@
 
 #define CLAPACK_INTERFACE
 
-#define DLANGE(X1,X2,X3,X4,X5,X6)       LAPACK(dlange,DLANGE)(LAPACK_COL_MAJOR,X1,X2,X3,X4,X5)
+
+extern "C" {
+    double dlange(const char&,const int&,const int&,double*,const int&,double*);
+}
+
+#define DLANGE dlange
 
```

Comment:
Actually dlange is in the MKL even in older versions (but not with the c++ header of lapacke), I had troubles to understand, but what we do for all other lapack functions is to call them directly through the c++ lapacke headers (mkl_lapacke.h), but there are no lapacke_dlange in MKL version 2013 (whereas there is such a signature in MKL 2017). 
so this fixes the problem, by calling the dlange provided (in all versions) in the mkl_lapack.h.
If you want, we can wait a look of @papadop for confirmation. but at least it now build on windows MKL 2013 AND uses the dlange lapack function.